.github/workflows/cicd.yml

```diff
@@ -1,6 +1,8 @@
 name: Syrupy CICD
 
-on: [push]
+on:
+  - push
+  - pull_request_target
```

Comment:
Confirmed this will trigger the job twice:

![image](https://user-images.githubusercontent.com/1297096/101945341-0fce4980-3bbc-11eb-9648-bebf412a868a.png)